pkgs/top-level/all-packages.nix

```diff
@@ -9644,8 +9644,8 @@ with pkgs;
   ### DEVELOPMENT / GO
 
   # the unversioned attributes should always point to the same go version
-  go = go_1_24;
-  buildGoModule = buildGo124Module;
+  go = go_1_25;
+  buildGoModule = buildGo125Module;
 
   go_latest = go_1_24;
   buildGoLatestModule = buildGo124Module;
```

Comment:
According to https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/go.section.md#versioned-toolchains-and-builders-ssec-go-toolchain-versions, should we not update latest, too? 😅

Reply:
That already happened on main via https://github.com/NixOS/nixpkgs/pull/433239.

I've cherry-picked https://github.com/NixOS/nixpkgs/pull/433348/commits/6c1f32536a3f1a99d4ca773db7a3b4a1b33abffb so this PR makes sense on staging.

Reply:
I've triggered https://github.com/NixOS/nixpkgs/actions/runs/16937031331 which should solve this after a rebase.

Reply:
Thanks, rebased.